recipes/default.rb

```diff
@@ -1,16 +1,12 @@
-package 'apache2' do
-  package_name value_for_platform_family(:rhel => "httpd", :debian => "apache2")
+package 'a better top' do
+  package_name value_for_platform_family(:rhel => "htop", :debian => "atop")
   retries 3
   retry_delay 5
   action :install
 end
-package 'git' do
```

Comment:
You are trying to install `git`. Git is already installed. Even if this recipe worked, what did you achieve in doing it ?